src/code/z_en_item00.c

```diff
@@ -710,18 +716,18 @@ void func_8001E5C8(EnItem00* this, GlobalContext* globalCtx) {
         this->actor.world.pos.y += 20.0f;
     }
 }
-
-extern s32 D_80157D90;
-extern u32 D_80157D90_; // these must be defined separately for EnItem00_Update to match.
-extern s16 D_80157D94;
-
+/* The BSS in the function acted weird in the past. It is matching now but mighy cause issues in the future*/
```

Comment:
```suggestion

/* The BSS in the function acted weird in the past. It is matching now but mighy cause issues in the future*/
```

Reply:
mighy -> might, and id use // personally